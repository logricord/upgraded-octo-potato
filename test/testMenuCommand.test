# Verify that the program outputs the menu correctly.
m

# Terminate program
q