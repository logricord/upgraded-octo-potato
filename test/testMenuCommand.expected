Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Verify that the program outputs the menu correctly.
Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu):
# Terminate program
